sdk/maps/azure-maps-render/README.md

```diff
@@ -168,20 +175,24 @@ And also save the result to file as png.
 ```python
 from azure.maps.render import MapsRenderClient
 
-result = maps_render_client.get_map_static_image(img_format="png", center=(52.41064,4.84228))
-# Save result to file as png
-file = open('result.png', 'wb')
-file.write(next(result))
-file.close()
+maps_render_client = MapsRenderClient(credential=AzureKeyCredential(subscription_key))
+
+result = maps_render_client.get_map_static_image(
+    zoom=10,
+    bounding_box_private= [13.228, 52.4559, 13.5794, 52.629]
```

Comment:
Can you check why this is "bounding_box_private" instead of "bounding_box"?

Reply:
Actually, not sure why is private, does it make sense to change generated code or to use Autorest directives to override param name to boudning_box instead of bounding_box_private?